client/my-sites/plugins/plugin-list-header/style.scss

```diff
@@ -82,3 +83,69 @@
 	display: none;
 	flex-grow: 1;
 }
+
+.plugin-list-header-new {
+	&.section-header.card {
+		&:not( .is-bulk-editing ) {
+			@include break-xlarge() {
+				display: none;
+			}
+		}
+		&.is-bulk-editing {
+			margin-top: 8px;
+			margin-bottom: 0;
```

Comment:
Can we use RTL equivalent properties here? `margin-block-start` and `margin-block-end` for `margin-top` and `margin-bottom` respectively here?